test/lib/ansible_test/_data/collection_detail.py

```diff
@@ -0,0 +1,69 @@
+"""Retrieve collection detail."""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+import os
+import sys
+
+import yaml
+
+
+def read_manifest_json(collection_path):
+    """Return collection information from the MANIFEST.json file."""
+    manifest_path = os.path.join(collection_path, 'MANIFEST.json')
+
+    if not os.path.exists(manifest_path):
+        return None
+
+    try:
+        with open(manifest_path) as manifest_file:
+            manifest = json.load(manifest_file)
+
+        collection_info = manifest.get('collection_info') or dict()
+
+        result = dict(
+            version=collection_info.get('version'),
```

Comment:
I just realized that the version lookup here (and below) should be passing the version through `LooseVersion` to make sure it's valid. Otherwise we'll get a traceback later when passing a bogus version to `pylint` or `validate-modules`.

Reply:
That reminds me that for collection versions, we should rather use `semantic_version.Version`  from https://pypi.org/project/semantic-version/ instead of `LooseVersion`.